### ISSUE TITLE
Examples not working

### ISSUE THREAD

**Issue by SneakBug8** [inkle/ink#287]
![2017-03-17 10_59_46-new issue inkle_ink - opera](https://cloud.githubusercontent.com/assets/11244723/24040390/1cceedb4-0b1a-11e7-8d40-64d5d2de0ded.png)

**Comment by joningold** [comment]
That's working as intended. The flow has run out and it needs a DONE or
END, but that's outside the scope of that example.

Jon

**Comment by SneakBug8** [comment]
Thanks for fast answer. Also, can I build game for PC without Unity?

**Comment by joningold** [comment]
No; though you can do File > Export for web to make a playable-in-browser
page.

cheers
jon
